src/command.ts

```diff
@@ -0,0 +1,23 @@
+import * as exec from '@actions/exec';
+import * as split from 'argv-split';
+import * as sub from 'substituter';
+
+export class Command {
+    static async execute(ocPath: string, args: string): Promise<number> {
+        if (!ocPath) {
+            return Promise.reject('Unable to find oc bundle');
+        }
+
+        const cmdArgs = await Command.prepareOcArgs(args);
+        return await exec.exec(`${ocPath} ${cmdArgs}`);
```

Comment:
Same I don't think you need Promise or I'm missing the await concept

Reply:
When using an async method you need to wrap the result in a Promise object or the code won't compile to js. I could remove the await before exec.exec without any error from the compiler but it will result in some bad behavior during execution. I'm not sure i got what you're trying to say.